app/views/layouts/application.html.erb

```diff
@@ -23,6 +23,16 @@
     <%= render 'header/header' %> 
     <section>
       <%= yield %>
+        <p class="navbar-text float-right">
```

Comment:
# Frontend review

- [ ] I couldn't find any link or reference to the frontend project.  In your next review request, kindly add a link to the frontend project on your backend readme file so a proper review can be carried out.